app/assets/src/components/views/discovery/DiscoverySidebar.jsx

```diff
@@ -159,9 +159,13 @@ export default class DiscoverySidebar extends React.Component {
             );
           })}
         </div>
-        <div className={cx(cs.dateHistogram)}>
+        <div
+          className={cx(cs.histogramLabels, dateKeys.length < 3 && cs.evenly)}
+        >
           <div className={cx(cs.label)}>{firstDate}</div>
-          <div className={cx(cs.label)}>{lastDate}</div>
+          {firstDate !== lastDate && (
+            <div className={cx(cs.label)}>{lastDate}</div>
```

Comment:
No need cx here... (neither in old code above)

Reply:
👍